src/example-relay/src/components/Navbar.js

```diff
@@ -105,6 +107,18 @@ function Navbar() {
   );
 }
 
+const brandAnimation = sx.keyframes({
+  [0]: {
+    transform: 'translateX(-300px) rotate(0deg)',
+  },
+  [80]: {
+    transform: 'translateX(10px) rotate(-5deg)',
```

Comment:
Maybe the rotate is too much here 🤷‍♂️

Reply:
I'd argue that the whole `brandAnimation` is a bit too much. 😛 How about creating test fixtures inside SX instead? They should be there anyway in my opinion.

Reply:
Sure, I can remove it entirely 👍 

It was just nice to _prove_ that it works ;)